packages/react-flicking/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@egjs/react-flicking",
-  "version": "0.0.4",
+  "version": "0.0.6",
```

Comment:
Why it isn't `0.0.5`?

Reply:
@WoodNeck 
 It seems that version 0.0.5 was missing when push was previously pressed.

This version has already been distributed.